playbooks/roles/mattermail/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+- name: create mattermail group
+  group:
+    name: mattermail
+
+- name: create mattermail user
+  user:
+    name: mattermail
+    group: mattermail
+    home: "{{ MATTERMAIL_HOME }}"
+    system: yes
+
+- name: Install git and golang packages
+  apt:
+    name:
+      - git
+      - golang
+    update_cache: yes
+
+- name: build Mattermail binary
+  command: go get {{ MATTERMAIL_REPO }}
+  args:
+    creates: "{{ MATTERMAIL_HOME }}/bin/mattermail"
+  become_user: mattermail
+  environment:
+    GOPATH: "{{ MATTERMAIL_HOME }}"
+
+- name: create config
+  template:
+    src: config.json
+    dest: "{{ MATTERMAIL_HOME }}/config.json"
+  become_user: mattermail
+
+- name: create data dir
+  file:
+    path: "{{ MATTERMAIL_HOME }}/data"
+    state: directory
+    owner: mattermail
+    group: mattermail
+
+- name: create systemd service
+  template:
+    src: mattermail.service
```

Comment:
nit: same as above, use the .j2 extension

Reply:
👍